rails/app/helpers/form_helper.rb

```diff
@@ -11,18 +11,17 @@ def submit(value = nil, options = {})
   def strada_form_with(*, **options, &block)
     options[:html] ||= {}
     options[:html][:data] ||= {}
-    options[:html][:data] = options[:html][:data].merge(strada_form_data)
-
+    options[:html][:data] = options[:html][:data].merge(strada_form_data(options))
     options[:builder] = StradaFormBuilder
 
     form_with(*, **options, &block)
   end
 
   private
 
-  def strada_form_data
+  def strada_form_data(options)
     {
-      controller: "bridge--form",
+      controller: "#{options.dig(:data, :controller)} bridge--form",
       action: "turbo:submit-start->bridge--form#submitStart turbo:submit-end->bridge--form#submitEnd"
```

Comment:
```suggestion
      action: "#{options.dig(:data, :action)} turbo:submit-start->bridge--form#submitStart turbo:submit-end->bridge--form#submitEnd"
```
I think it should be extended also to the actions in case some custom actions are defined in addition to the bridge--form default actions